tools/c7n_index/c7n_index/metrics.py

```diff
@@ -128,35 +145,47 @@ def get_indexer(config, **kwargs):
 @indexers.register('es')
 class ElasticSearchIndexer(Indexer):
 
-    def __init__(self, config, **kwargs):
+    def __init__(self, config):
         self.config = config
-        self.es_type = kwargs.get('type', 'policy-metric')
-
-        host = [config['indexer'].get('host', 'localhost')]
+        # self.es_type = kwargs.get('type', 'policy-metric')
+        host = [config['indexer'].get('host')]
         kwargs = {}
-        kwargs['connection_class'] = RequestsHttpConnection
+        kwargs['connection_class'] = elasticsearch.RequestsHttpConnection
 
         user = config['indexer'].get('user', False)
         password = config['indexer'].get('password', False)
         if user and password:
             kwargs['http_auth'] = (user, password)
 
-        kwargs['port'] = config['indexer'].get('port', 9200)
```

Comment:
Why remove the default value of `9200`?

Reply:
same as `localhost`, `port` is required in the schema, but will add this back.